server/pages/index.jade

```diff
@@ -240,7 +240,7 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 
 		script.
 			(function() {
-				if ( window.console ) {
+				if ( window.console && 'development' !== window.configData.env ) {
```

Comment:
Is it worth guarding this check with the following to avoid errors where `configData` is undefined? ... 
```js
window.console && window.configData && 'development' !== window.configData.env
```

Reply:
Yes, otherwise we might brea Calypso 😉

Reply:
Good catch, thanks! Updated :)